test/loopback.test.js

```diff
@@ -698,7 +698,7 @@ describe('loopback', function() {
       expect(methodNames).to.include('hello');
     });
 
-    it('warn remote methods defined via configureModel after child model was ' +
+    it('remote methods defined via configureModel after child model was ' +
```

Comment:
`it('does not inherit remote methods defined via ...')`

Remember, the test name should read like an English sentence.